frc/4. Autonomous/1. Overview.md

```diff
@@ -1,7 +1,42 @@
 # Overview
 
+## Conceptual Aspect
 
-reiterate that we want to score and use this time as efficiently as possible. 
+As mentioned before, autonomous is a 15-second period in the beginning of a match where robots can move independent of driver control to score and support their alliance. 
 
-Our scoring actions will vary year to year, whether it be revving up a shooter wheel, or concurrently raising an elevator and running an intake, all ultimately to score game pieces--however, the bulk of the work for auto is *pathing*, namely planning drivetrain trajectories and movement. 
+**MAIN IDEA: To time-efficiently score as much as possible.**
 
+Our scoring tasks will vary year to year, whether it be shooting or placing a game piece. However, the bulk of the work for us as programmers in auto is *pathing*, namely planning drivetrain trajectories and movement. We'll see more of this in the next few sections.
+
+## Programming Aspect
+
+A broad description of what we do: we organize the commands and subsystems already written out such that they form an autonomous *routine*. Since we generally are trying to accomplish a difficult task independent of driver control, we usually need to augment our robot capability. This might mean adding closed-loop control to various subsystems, or in other words, just being able to *control our robot better.* Having a well-tuned robot makes everything a lot easier, especially for autonomous.
+
+Auto routines will be programmed in the form of commands, and therefore should go in an /auto folder under /commands.
+
+## Examples
+
+Here are a few examples of really riveting, top-tier autonomous robot performances.
+
+![254 Barrage Reveal](img&gif/254barrageReveal.gif) [Source](https://www.youtube.com/watch?v=aFZy8iibMD0)
+
+Team 254 (The Cheesy Poofs) had a very consistent autonomous in 2014 that allowed them to score the maximum amount of game pieces.. The trajectory is a simple s-shaped curve, but the complexity lies in their ability to hold the three balls and shoot them both quickly and reliably.
```

Comment:
I would personally avoid saying "simple s-shaped curve" since at the time, it was pretty revolutionary and for teams that have been doing segmented path following (us!) even an s-shaped curve can be difficult